File: examples/client/javascript/fiddle_client.js
const io = require('socket.io-client')
const port = process.env.PORT || 5000;

const socket = io('http://localhost:' + port, {auth: {token: 'my-token'}});

socket.on('connect', () => {
  console.log(`connect ${socket.id}`);
});

socket.on('disconnect', () => {
  console.log(`disconnect ${socket.id}`);
});

socket.on('hello', (a, b, c) => {
  console.log(a, b, c);
});

File: examples/client/sync/fiddle_client.py
import socketio

sio = socketio.Client()


@sio.event
def connect():
    print('connected to server')


@sio.event
def disconnect(reason):
    print('disconnected from server, reason:', reason)


@sio.event
def hello(a, b, c):
    print(a, b, c)


if __name__ == '__main__':
    sio.connect('http://localhost:5000', auth={'token': 'my-token'})
    sio.wait()
